demo/views.py

```diff
@@ -96,3 +99,38 @@ def select_random_six_demo_images(images_list):
         if i.startswith(prefixes):
             demo_images = select_random_six_demo_images(images_list)
     return demo_images
+
+
+def upload_image_using_url(request):
+    if request.method == "POST":
+        try:
+            socketid = request.POST.get('socketid', None)
+            image_url = request.POST.get('src', None)
+            demo_type = request.POST.get('type')
+
+            if demo_type == "dba":
+                dir_type = constants.DBS_CONFIG['image_dir']
+
+            img_name = os.path.basename(urlparse(image_url).path)
+            response = requests.get(image_url, stream=True)
+
+            if response.status_code == 200:
+                random_uuid = uuid.uuid1()
+                output_dir = os.path.join(dir_type, str(random_uuid))
+
+                if not os.path.exists(output_dir):
+                    os.makedirs(output_dir)
+
+                img_path = os.path.join(output_dir, str(img_name))
+                with open(os.path.join(output_dir, img_name), 'wb+') as f:
+                    f.write(response.content)
+
+                img_path =  "/" + "/".join(img_path.split('/')[-5:])
+                
+                return JsonResponse({"file_path": img_path})
+            else:
+                return HttpResponse("Please Enter the Correct URL.")
```

Comment:
Can you please replace the `HttpResponse ` with `JsonResponse ` and do the same for other return statements ?